### ISSUE TITLE
Breaks with Eslint 9

### ISSUE THREAD

**Issue by ajslater** [dustinspecker/eslint-plugin-no-use-extend-native#153]
Using eslint v9.10.0

On macOS with homebrew I get some warnings, but eslint runs

```sh
(node:51025) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
    at emitExperimentalWarning (node:internal/util:268:11)
    at ModuleLoader.jsonStrategy (node:internal/modules/esm/translators:431:3)
    at callTranslator (node:internal/modules/esm/loader:436:14)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:442:30)
```

On Ubuntu 2404 it actually breaks.

```sh
SyntaxError: Unexpected token 'with' at ModuleLoader.moduleStrategy (node:internal/modules/
```

Removing the eslint-plugin-no-use-extend-native package removes the warnings from macOS and lets eslint run on Ubuntu.

**Comment by lishaduck** [comment]
This has nothing to do with ESLint 9, but rather with ancient versions of node. This package uses [import attributes](https://nodejs.org/api/esm.html#json-modules).

**Comment by silverwind** [comment]
Yup, this package works fine in eslint 9.
